CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# 1.6.0
+## General
+
+Added
+* Allow customization of service annotations in Helm values ([#738] thanks @mbovo)
+* Allow arbitrary labels and annotations to be added to all workloads or selectively ([#742] thanks @grumps)
+
+Fixed
+* script: use --update=none for mv in publish-charts ([#731])
+
+Misc
+* Numerous dependency upgrades and documentation fixes ([#732], [#738], [#744])
```

Comment:
```suggestion
* Upgrade dependencies ([#732], [#744])
```

I don't actually don't see a separate change that is updating the documentation? I see that #742 is updating docs but I don't think we need to call it out separately from that change.

If you want to reference the changes in this PR, please include this PR #749 in the changelog.